.github/workflows/bazel.yml

```diff
@@ -350,6 +354,30 @@ jobs:
             cd /workspaces/magma
             set -euo pipefail
 
+            # Required for bazel-diff to perform git checkout.
```

Comment:
We now use the bazel diff in three places - do you think this can be modeled more DRY? E.g., pull more parts into the separate scripts.

Reply:
This will be handled in a separate ticket https://github.com/magma/magma/issues/14553